src/View/Widget/BasicWidget.php

```diff
@@ -72,6 +74,24 @@ public function render(array $data, ContextInterface $context): string
         $data['value'] = $data['val'];
         unset($data['val']);
 
+        $fieldName = $data['fieldName'] ?? null;
+        if (
+            $data['type'] === 'number'
+            && !isset($data['step'])
+            && $fieldName
+        ) {
+            $data = $this->setStep($data, $context, $data['fieldName']);
```

Comment:
Should this be `$fieldName` instead of `$data['fieldName']`?

Reply:
Yup, done.